includes/admin/dashboard-widgets.php

```diff
@@ -149,16 +149,16 @@ function edd_load_dashboard_sales_widget( ) {
 		<div style="clear: both"></div>
 		<?php do_action( 'edd_sales_summary_widget_after_stats', $stats ); ?>
 		<?php
-		$payments = edd_get_payments( array( 'number' => 5, 'status' => 'complete' ) );
+		$payments = edd_get_orders( array( 'number' => 5, 'status' => 'complete' ) );
```

Comment:
I think we should update this to `$orders` to avoid any possible confusion for what is returned.